cnf-tests/.konflux/README.md

```diff
@@ -0,0 +1,40 @@
+**Update RPMs dependencies**
+
+In this project (cnf-tests) we are following the hermetic builds flow to install RPM dependencies according to the Product Security requirments. For more info on why this is needed please follow the info in the following link:
+https://konflux.pages.redhat.com/docs/users/faq/hermetic.html.
+
+
+To delete no longer needed dependencies you need to:
+1. remove the binary name from `rpms.in.yaml` and from `Dockerfile`.
+2. remove the matching section for this binary from `rpms.lock.yaml`.
```

Comment:
If preferred, you could re-run `rpm-lockfile-prototype` to regenerate it from `rpms.in.yaml`, instead of manually editing it.

Reply:
you're right, I mentioned this option in https://github.com/openshift-kni/cnf-features-deploy/pull/2167/files/a46c342c9b845c1b66902ed9046e49e8c193c036#diff-c08a78341751a7ef28dd58f831921ab6668ce1c20a4327fd2b1417baa99bef13R13